api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
Watch copyrights -- `Simon Fraser University Library` should be `Simon Fraser University`, and the end dates should be `-2017`.